source/documentation/strategy-and-principles/techops-shared-responsibility-model.html.md

```diff
@@ -82,6 +82,8 @@ Specifically, TechOps will enforce the established best practice of "two-eyes" (
 
 ### Responding to security events and incidents and acting on vulnerabilities
 
+As a general rule, TechOps will adopt the service team's existing incident management and support processes, until a consistent, cross-TechOps model has been established. 
```

Comment:
No problem at all with this content. For efficiency's sake I would say you can exclude `As a general rule, `. Also, are there any timescales attached to the cross-TechOps model? If not then no problem